src/Worker.php

```diff
@@ -241,4 +243,44 @@ public function terminate(): void
     {
         $this->dispatchEvent($this->app, new WorkerStopping($this->app));
     }
+
+    /**
+     * Handle Incoming WebSocket Messages
+     */
+    public function handleWebSocketMessage(\Swoole\WebSocket\Server $server, \Swoole\WebSocket\Frame $frame)
+    {
+        \Laravel\Octane\CurrentApplication::set($sandbox = clone $this->app);
```

Comment:
Pretty much copied from `handleTick`/`handleTask`, does it make sense? Should it be doing these same steps for incoming WebSocket messages/disconnect?